lib/windows/winlean.nim

```diff
@@ -1059,6 +1059,28 @@ else:
                         lpNumberOfEventsRead: ptr cint): cint
        {.stdcall, dynlib: "kernel32", importc: "ReadConsoleInputW".}
 
+const
```

Comment:
These are all terminal specific and so should be in terminal.nim.

Reply:
Done.